.buildbot.sh

```diff
@@ -83,10 +83,6 @@ cd ../../
 PATH=${YKB_YKLLVM_BIN_DIR}:${PATH} clang-format --version
 # Check C/C++ formatting using xtask.
 PATH=${YKB_YKLLVM_BIN_DIR}:${PATH} cargo xtask cfmt
-# This is used to check clang-tidy output, but the dirty submodule from building
-# ykllvm is also shown.
-# FIXME: Add build/ to .gitignore in ykllvm
-git diff --exit-code --ignore-submodules
```

Comment:
Does `cargo xtask cfmt` exit non-zero if there were formatting changes then?

Reply:
What's that got to do with `git diff`?

Reply:
I really don't get how `git diff` on its own in CI can do anything useful.

Reply:
How can there be unstaged changes in CI?

Reply:
`cargo xtask cfmt` does in-place changes if formatting is required.

Reply:
OK so I was completely mislead by the comment. Fixed in 325bf652.

[I also removed the `clang-format` check which is pointless and also very slightly mislead me.]

Reply:
OK.

I'm not 100% sure why the submodule would be dirty, but let's assume it could happen :)

Reply:
Yeah, but the `ykllvm` dir is explicitely excluded from the formatting run:
https://github.com/ykjit/yk/blob/9f01171281f4d5114a0fa7d12363f9a81102f6b6/xtask/src/main.rs#L10

Reply:
Well it doesn't hurt to exclude the submodule I guess.